lib/autoyast.pm

```diff
@@ -338,4 +350,160 @@ sub create_report {
     return "$error_summary\n\n$expressions_summary\n";
 }
 
+=head2 detect_profile_directory
+
+ detect_profile_directory(profile => $profile, path => $path)
+
+ Try to detect profile directory (autoyast_opensuse/, autoyast_sle{12,15}/, autoyast_sles11/)
+ and returns its path.
+ # TODO: autoyast_{caasp,kvm,qam,xen}
```

Comment:
i guess you dont need # here :)

Reply:
there is section RESTRICTION for Perlpod to add those limitations, but let's not over-complicate it, I'm fine with remove `#` it was just the habit for other languages when those metadata helps with doc generation.